packages/annotations/src/layer-definition.ts

```diff
@@ -202,6 +202,15 @@ export const annotationServiceDefinition: ILayerDefinition = {
       editable: true,
       domain: null,
       defaultValue: null
+    },
+    {
+      name: "value",
+      type: "esriFieldTypeInteger",
+      alias: "value",
+      nullable: true,
+      editable: true,
+      domain: null,
+      defaultValue: null
```

Comment:
just to be explicit, this creates a new `value` field in hosted feature services that have yet to be created. i'm not sure what we can/should do about the ones that are already 'in the wild'.

Reply:
we should probably create a migrate function

Reply:
to be clear, I don't necessarily mean that has to happen in this PR.

Reply:
that's what i assumed.